init.ts

```diff
@@ -109,6 +109,28 @@ await Deno.writeTextFile(
   GITIGNORE,
 );
 
+const DOCKERFILE_TEXT = `
+FROM denoland/deno:1.35.0
```

Comment:
This should match the version of Deno that is currently running.

Reply:
I'll take care of this and re-ping, already know how to take care of it 👍